assets/scss/common/_custom.scss

```diff
@@ -18,3 +18,7 @@
 .image-hover:hover {
     opacity: 1;
 }
+[data-bs-theme="dark"] .image-hover {
```

Comment:
It feels inconsistent to only have a hover effect on the logo for one theme. 

Could you either revert this or find some other way to react to user input? For instance, having it glow a slight amount, with the glow expanding on hover would work well, I'd think.